lib/web_ui/dev/safari.dart

```diff
@@ -23,37 +23,50 @@ class Safari extends Browser {
 
   static String version;
 
+  static bool mobileBrowser;
```

Comment:
nit: this should be called `isMobileBrowser`.

https://dart.dev/guides/language/effective-dart/design#prefer-a-non-imperative-verb-phrase-for-a-boolean-property-or-variable

Reply:
done.